nexus/auth/src/authn/silos.rs

```diff
@@ -68,57 +62,6 @@ pub enum IdentityProviderType {
     Saml(SamlIdentityProvider),
 }
 
-impl IdentityProviderType {
```

Comment:
This method was moved into `nexus/db-queries/src/db/datastore/identity_provider.rs` - it doesn't need to be defined here, and is tightly coupled with the datastore, not auth logic.